.github/scripts-windows/run-tests-python3.cmd

```diff
@@ -1,3 +1,3 @@
 cd runtime-testsuite
-mvn -Dantlr-python3-exec="C:\Python310\python.exe" -Dtest=python3.** test
+mvn -Dantlr-python3-exec="%PYTHON_HOME%\python.exe" -Dtest=python3.** test
```

Comment:
I disapprove this change.
This runs in an always on docker instance with both Python2 and Python3 installed
The python version at PYTHON_HOME is unknown